frontend/components/DocumentFilters/DocumentFilters.jsx

```diff
@@ -140,9 +166,8 @@ export default class DocumentFilters extends Component {
     const {updateUrlParams} = this.props
 
     let validFilters = filters.filter(entry => {
-      let isValid = Object.keys(entry).filter(key => {
-        return Object.is(entry[key], null)
-      })
+      let isValid = Object.keys(entry)
+        .filter(key => Object.is(entry[key], null))
```

Comment:
In the case you linked to, the alternative is to check that an object is equal to one or another value. This check is what `Object.js` is designed for. 

You may have overlooked this, but there are many methods we're using that aren't supported by IE.